index.js

```diff
@@ -117,7 +130,7 @@ function transformSource (runner, config, source, map, callback) {
 function addDependencies (loader, paths, callback) {
   var remaining = paths.length
 
-  if (remaining === 0) callback(null)
+  if (remaining === 0) { callback(null) }
```

Comment:
Please remove changes here and respect existing .eslintrc config.

Reply:
Gosh, I didn't see the `.eslintrc`! I'll remove the `.jshintrc` (and these changes). Just didn't want my editor highlighting every line with linting errors while I was trying to work. :-P